docs/src/pages/quick-start.md

```diff
@@ -26,14 +23,74 @@ function App() {
   return (
     // Provide the client to your App
     <QueryClientProvider client={queryClient}>
-      <Todos />
     </QueryClientProvider>
   )
 }
+```
+
+Here we are creating a client by creating a new instance of the `QueryClient` class.
+A query client can be used to deal with things like cache.
+
+
+Next, we pass the instance to the `QueryClientProvider` which must be a parent to all components that use `react-query`.
+It connects the queryClient with your application. This is quite similar to the `Provider` component that is given by `redux`.
+
+Let's now create a todo component:
+
+```js
+import {
+  useQuery,
+  useQueryClient,
+  QueryClient,
+  QueryClientProvider,
+} from 'react-query'
+import { getTodos, postTodo } from '../my-api'
+
+// ...app component
+
+function Todos() {
+  // Access the client
+  const queryClient = useQueryClient();
+
+  // Queries
+  const query = useQuery('todos', getTodos)
+
+  return (
+    <div>
+      <ul>
+        {query.data.map(todo => (
+          <li key={todo.id}>{todo.title}</li>
+        ))}
+      </ul>
+    </div>
+  )
+}
+
+render(<App />, document.getElementById('root'))
+```
+
+In the `Todo` component, we start by accessing the queryClient we defined earlier using the `useQueryClient` hook.
+A `query` listens to a source of data, in our case, the `getTodos` method.
```

Comment:
this makes it sound like we're listening to the function `getTodos`, which is not true. We're listening to the query key, not the function. It gives the impression that the functional reference must be stable, which it is here, but it doesn't have to be.